cmd/ocsp-updater/main.go

```diff
@@ -153,20 +153,20 @@ func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Ti
 	now := updater.clk.Now()
 	maxAgeCutoff := now.Add(-updater.ocspStaleMaxAge)
 
-	certStatusFields := "cs.serial, cs.status, cs.revokedDate, cs.notAfter"
-	if features.Enabled(features.StoreIssuerInfo) {
-		certStatusFields += ", cs.issuerID"
-	}
 	_, err := updater.dbMap.Select(
 		&statuses,
-		fmt.Sprintf(`SELECT
-				%s
+		`SELECT
```

Comment:
I think the check in generateResponse can also be removed now?

https://github.com/letsencrypt/boulder/blob/deprecate-storeissuerinfo-ocsp-updater/cmd/ocsp-updater/main.go#L213-L222